planning_through_contact/simulation/controllers/hybrid_mpc.py

```diff
@@ -169,6 +182,11 @@ def _setup_QP(
 
         # Control constraints
         mu = self.dynamics_config.friction_coeff_slider_pusher
+        # Control limits:
+        MAX_MAGNITUDE = 0.3
```

Comment:
Are we sure these constants are not too small for the generated trajectory? We should at least make sure we don't forget we set these here, in case they cause problems in the future.

Reply:
Last I checked these values were giving good (better?) results for the generated trajectories, but I've just added these values to the `HybridMpcConfig` so we can more easily change them and know that they can be changed